composer.json

```diff
@@ -13,16 +13,21 @@
     "require": {
         "php": "8.3.*",
         "algolia/algoliasearch-client-php": "^3.3",
+        "doctrine/collections": "^2",
         "doctrine/dbal": "^4.0",
         "doctrine/inflector": "^2.0",
         "doctrine/orm": "^3.2",
         "doctrine/static-website-generator": "^2.1",
         "erusev/parsedown": "^1.7",
         "knplabs/github-api": "^3.9",
+        "league/flysystem": "^1",
+        "league/tactician": "^1",
```

Comment:
I don't understand why this is supposed to be a dependency.

Reply:
@SenseException because it is mentioned here: https://github.com/doctrine/doctrine-website/blob/69a03f9b49164a937a24f8f8b544196a2a0b24d2/lib/Docs/RST/GuidesParser.php#L11